src/SearchResults.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Moment from "react-moment";
+import moment from "moment";
+
+const SearchResults = props => {
+  let isClicked = false;
+  // function theRowisclicked(){
+  //   if (!isClicked){
+  //     isClicked=true;
+  //     e.target.classList.add("highLight")
+
+  //   }
+  //   else{
+  //     isClicked=false;
+  //     e.target.classList.remove("highLight")
+  //   }
+  // }
+  return (
+    <div className="table-responsive-sm">
+      <table className="table table-bordered">
+        <thead className="table-primary fontColor">
+          <tr
+            onClick={e => {
+              if (!isClicked) {
+                isClicked = true;
+                e.target.classList.add("highLight");
+              } else {
+                isClicked = false;
+                e.target.classList.remove("highLight");
+              }
+            }}
+          >
+            <th scope="col">#</th>
+            <th scope="col">Title</th>
+            <th scope="col">Firstname</th>
+            <th scope="col">Surname</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room id</th>
+            <th scope="col">Check in date</th>
+            <th scope="col">Check out date</th>
+            <th scope="col">Duration of stay </th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map(result => {
+            return (
+              <tr
+                onClick={e => {
+                  if (e.currentTarget.classList.contains("highLight")) {
+                    e.currentTarget.classList.remove("highLight");
```

Comment:
That's clever solution🙂 However it's not the way it should be done in React. I would suggest following these steps:
1. Extract code from lines 47 to 70 to a new `TableRow` component - you can have a look at the solution to lesson 1 exercises and do the same.
2. Inside that new `TableRow` component declare a new state `isSelected` with the corresponding setter function `setIsSelected` and initially set the value to `false`.
3. Pass a function to the `onClick` handler that changes `isSelected` to the opposite value when clicked:
```
() => setIsSelected(!isSelected)
```
`!` operator will basically change `true` to `false` and `false` to `true`.
4. Then you can add a `className` attribute to your `<tr>` element and within a curly braces add a ternary operator that assigns a class name depending on the value of `isSelected`. For example if it's true - assign the class name with highlighted background - otherwise assign a different one or even null:
```
className={isSelected ? "highLight" : null}
```